src/pfe/portal/modules/LoadRunner.js

```diff
@@ -636,6 +645,26 @@ module.exports = class LoadRunner {
     this.emitCompleted();
   }
 
+  async postProcessProfilingData() {
+    /* Run the merge out of process as the profiling data could potentially be quite large. */
+    const profilingPath = path.join(this.workingDir + '/profiling.json');
+    const normalisedProfilingPath = this.project.getPathToProfilingTreeFile(this.metricsFolder);
+    const mergeCommand = `${process.execPath} /portal/scripts/mergeNodeProfiles.js ${profilingPath} ${normalisedProfilingPath}`;
+    log.debug(`Running: ${mergeCommand}`);
+    const mergeProcess = await exec(mergeCommand);
+    log.trace(mergeProcess.stdout);
+    log.trace(mergeProcess.stderr);
+    log.debug('Merging done');
```

Comment:
Nice debug

Reply:
That should result in an exception and triggering of the `catch` block.
I've realised the message from the summarising block below says `merging` so I'll push a fix for that.

Reply:
That should result in an exception and triggering of the `catch` block.
I've realised the message from the summarising block below says `merging` so I'll push a fix for that.